src/main/java/pipe/gui/petrinet/graphicElements/AnnotationNote.java

```diff
@@ -189,14 +189,8 @@ public boolean enableEditMode(boolean isFirstEdit) {
 
 		guiDialog.add(new AnnotationPanel(this));
 		guiDialog.setMinimumSize(new Dimension(300, 200));
+		
 		// Make window fit contents' preferred size
-		guiDialog.addWindowListener(new WindowAdapter() {
```

Comment:
is some other code making sure the annotation is not added to the net if you don't put in some text?